pkg/provider/operators.go

```diff
@@ -95,9 +100,12 @@ func (op *Operator) SetPreflightResults(env *TestEnvironment) error {
 	ctx = logr.NewContext(ctx, logger)
 
 	check := plibOperator.NewCheck(bundleImage, indexImage, oc.KubeConfig, opts...)
-	results, err := check.Run(ctx)
-	if err != nil {
-		return err
+	var results plibRuntime.Results
```

Comment:
What is a rational to have more lines? `res, err := check.Run(ctx)` is cool.

Reply:
Fixed!